opennlp-tools/src/main/java/opennlp/tools/ml/maxent/GISTrainer.java

```diff
@@ -228,6 +228,7 @@ public GISModel trainModel(ObjectStream<Event> eventStream, int iterations, int
     params.put(GIS.ITERATIONS_PARAM, Integer.toString(iterations));
     params.put(GIS.CUTOFF_PARAM, Integer.toString(cutoff));
     indexer.init(params, new HashMap<>());
+    indexer.index(eventStream);
```

Comment:
It is a bit scary that there is no failing test for this. We should have one to make sure this code path works.